### ISSUE TITLE
Feature: Colorblind accessibility options for colored lasers

### ISSUE THREAD

**Issue by Critfish** [ChaosInitiative/Portal-2-Community-Edition#516]
## Which component should be improved?
Render, Entity, Gameplay, UI/UX

## Describe your suggestion
Some form of colorblind mode for colored lasers. They're a great testing element, but anybody with colorblindness will struggle to differentiate the different laser types.

## Expected result
There are several ways you could go about solving this issue. HUD overlays could work, though that could look messy. Placing symbols on the emitters/cubes/receivers might work, but it'd still leave the lasers themselves unable to be identified. Perhaps a "shape" projected along the laser beam itself could work, without looking too visually noisy, though from longer distances, it may still not work properly.

**Comment by SpyceTewan** [comment]
Having similar laser colors isn't a good design principle in general. As a designer you should factor this in, but i don't know how something like this would be changed on a code-level

Also for opening suggestions, use the discussions tab and only open an issue if you're actually certain what the feature should look like

**Comment by seagemgames** [comment]
If this is ever added, I feel like there should be a specific menu for accessibility settings that's easy to find in the options. I hate when people are excluded because they are different, and having more options for colourblindness is a good start.

**Comment by ozxybox** [comment]
A fast, perhaps not pretty, solution to this would be using Source's built in "Noise" function for beams (where the beams move and look more like electricity). This might defeat the idea of it being a laser though...

**Comment by ozxybox** [comment]
Marked as medium rather than low, as colored lasers would be very frustrating to those with colorblindness.
